native/objcexport-header-generator/src/org/jetbrains/kotlin/backend/konan/objcexport/StubRenderer.kt

```diff
@@ -119,6 +122,14 @@ object StubRenderer {
         append(';')
     }
 
+    private fun renderNativeEnumType(nativeEnum: ObjCNativeEnum): String = buildString {
```

Comment:
`ObjCNativeEnum` and its rendering are trying to do too much.
Other stubs are pretty straightforward: they encapsulate syntax structures and are simply rendered without making decisions (except for kdoc things).
But `ObjCNativeEnum` is more high-level, and rendering makes some decisions, e.g.:
* Selects the underlying type
* Selects values
* Adds attributes (`NS_SWIFT_NAME`)

Reply:
I have tried to align this more with the rest of the code here.